website/static/js/home-page/quickProjectSearchPlugin.js

```diff
@@ -539,6 +539,29 @@ var QuickSearchProject = {
     }
 };
 
+function ancestorName(node, ancestor, title) {
+    var name;
+    var request = lodashGet(node, 'embeds.' + ancestor + '.data.attributes.title', '');
```

Comment:
This is just a little harder to follow than I'd like, and it's probably because of variable naming, especially `request` and `title`, although `ancestor` might also be more descriptive, such as `ancestorDescriptor` or something (`ancestor` sounds as though it might be an object, for example).